README.md

```diff
@@ -49,6 +51,26 @@ the image first.
 
 - When confirmed, click on the `Flash` button to start flashing procedure.
 
+### bmaptool
+
+If you prefer a command-line tool, then you should use bmaptool, just install it with
+
+```
+sudo apt install bmap-tools
+```
+
+then run with
+
+```
+sudo bmaptool copy path/to/image/example.wic.gz /dev/yourSDcard
```

Comment:
@WiktorG351 it is better to use `bmaptool copy --bmap ....wic.bmap ....wic.gz /dev/sdX` using bmap file speed up the whole process.

Also it would be nice to point here how to get information about which /dev/* we should use.

Reply:
fixed in https://github.com/3mdeb/meta-rte/blob/734d074babc8652f7e6ba7f7ada1e5a296b60f0b/README.md?plain=1#L80